Backpack-SwiftUI/SearchInputSummary/Classes/BPKSearchInputSummary.swift

```diff
@@ -35,22 +36,33 @@ public struct BPKSearchInputSummary: View {
     @Binding private var text: String
     @FocusState private var focused: Bool
     private let placeholder: String
-    private let inputPrefix: InputPrefix
+    private let inputPrefix: InputPrefix?
```

Comment:
wasnt this a requirement that the search input summary component will always have a prefix?

Reply:
Yes, it was before
Now it should be optional :)

Reply:
ahh okay, i just found the figma file. alright!